modules/source/mail/config/mail.xml

```diff
@@ -34,7 +38,7 @@
 			store-uri="${protocol}://${username:}:${password:}@${host}:${port}/${folder}"
```

Comment:
We should assume that users won't have access to edit the file and also different stream instances may want to use different properties.  I guess the location isn't as important - if we want to reduce duplication for shared config.  I guess we have a few examples of that already.  Adding module options for the current Boolean props and allowing to pass in additional props should be added.